build.cake

```diff
@@ -375,41 +383,35 @@ Task("PackageConsole")
     .IsDependentOn("CreateImage")
     .Does(() =>
     {
-        var currentImageDir = IMAGE_DIR + "NUnit-" + packageVersion + "/";
-
         CreateDirectory(PACKAGE_DIR);
 
         NuGetPack("nuget/runners/nunit.console-runner.nuspec", new NuGetPackSettings()
         {
             Version = packageVersion,
-            BasePath = currentImageDir,
+            BasePath = CURRENT_IMG_DIR,
             OutputDirectory = PACKAGE_DIR,
             NoPackageAnalysis = true
         });
 
         NuGetPack("nuget/runners/nunit.console-runner-with-extensions.nuspec", new NuGetPackSettings()
         {
             Version = packageVersion,
-            BasePath = currentImageDir,
+            BasePath = CURRENT_IMG_DIR,
             OutputDirectory = PACKAGE_DIR,
             NoPackageAnalysis = true
         });
 
         NuGetPack("nuget/runners/nunit.runners.nuspec", new NuGetPackSettings()
         {
             Version = packageVersion,
-            BasePath = currentImageDir,
+            BasePath = CURRENT_IMG_DIR,
             OutputDirectory = PACKAGE_DIR,
             NoPackageAnalysis = true
         });
     });
 
-// Unlike other tasks, this one does not depend on creation of the
-// image directory. I think this is the direction we want to go
-// for all the different packages, but it's for a separate change.
 Task("PackageChocolatey")
 	.Description("Creates chocolatey packages of the console runner")
-	.IsDependentOn("Build")
```

Comment:
This initially confused me - by running the 'Package' task, you ended up with different binaries being packaged for NuGet than for Chocolatey. That seemed dangerous - so I removed it for now. I think if we want this dependency, we need to add it for all packaging - but we should look at that as a separate issue.

Reply:
If you verified that, it maybe a Cake bug. Things depended on more than once should only run once. That's how every build system since make has worked. :-)

Reply:
I don't think so in this case, it's a based on the order of dependencies of the Package task. So, packaging currently runs
```
PackageNuGet ->PackageChoco -> Package Msi
```

Because only Choco depended on the build task, that evaluated to this:
```
PackageNuGet -> Build -> PackageChoco -> Package Msi
```

That meant NuGet packaging was done with the old binaries, then they were rebuilt, then choco packaging was done.

Reply:
I misunderstood. I thought the other packages also depended on build directly or indirectly. If we want packaging to __not__ build but use the latest build, then there should be no dependencies. If we want __any__ packaging to trigger a build, then each individual package should depend on build. If we want only the full package target to trigger a build, then that should have the dependency.

It's possible I created the inconsistency when I was experimenting with different ways to do this. I'm afraid you may find various half-done things from the point where I stopped working on the build because the only way to test some of them in CI was to commit them. 😞